app/_tutorials/add-rate-limiting-tiers-with-kong-gateway.md

```diff
@@ -38,57 +38,110 @@ tags:
 
 tools:
     - deck
-
-prereqs:
-    consumers:
-        - example-consumer
-    consumer-groups:
-        - example-consumer-group
 ---
 
-With consumer groups, you can define rate limiting tiers and apply them to subsets of application consumers.
+## Steps
+
+1. Create the Free, Basic, and Premium tier consumer groups:
+  ```yaml
+  _format_version: '3.0'
+  consumer_groups:
+  - name: Free
+  - name: Basic
+  - name: Premium
+  ```
+  Add this configuration to a `kong.yaml` file in a `deck_files` directory.
+
+1. Synchronize your configuration
 
-You can define consumer groups as tiers, for example:
+   Check the differences in your files:
+   ```sh
+   deck gateway diff deck_files
+   ```
 
-* A **gold tier** consumer group with 1000 requests per minute
-* A **silver tier** consumer group with 10 requests per second
-* A **bronze tier** consumer group with 6 requests per second
+   If everything looks right, synchronize them to update your Kong Gateway configuration:
+   ```sh
+   deck gateway sync deck_files
+   ```
+
+1. Create four consumers, one for each tier and one that won't be assigned to a consumer group tier:
   
-Consumers that are not in a consumer group default to the Rate Limiting advanced plugin’s configuration, so you can define tier groups for some users and have a default behavior for consumers without groups.
+  ```yaml
+  consumers:
+  - username: Amal
+    groups:
+    - name: Free
+  - username: Dana
+    groups:
+    - name: Basic
+  - username: Mahan
+    groups:
+    - name: Premium
+  - username: Rosario
+  ```
+
+  Apphend this to your `kong.yaml` file. 
+
+  In this tutorial, one consumer won't be assigned to a tier, which is a consumer group. Consumers that are not in a consumer group default to the Rate Limiting advanced plugin’s configuration, so you can define tier groups for some users and have a default behavior for consumers without groups.
 
-## Steps
+1. Synchronize your configuration
+
+   Check the differences in your files:
+   ```sh
+   deck gateway diff deck_files
+   ```
 
-1. Enable the Rate Limiting Advanced plugin on the consumer group:
-{% capture step %}
-  {% entity_example %}
-    type: plugin
-    data:
-      name: rate-limiting-advanced
-      config:
-        limit: 5
-        window_size: 30
-        window_type: sliding
-        retry_after_jitter_max: 0
- 
-    targets:
-        - consumer_group
-
-    variables:
-      'consumerGroupName|Id': Gold
-  {% endentity_example %}
-{% endcapture %}
-{{ step | indent: 3 }}
-
-    This configuration sets the rate limit to five requests (`config.limit`) for every 30 seconds (`config.window_size`).
+   If everything looks right, synchronize them to update your Kong Gateway configuration:
+   ```sh
+   deck gateway sync deck_files
+   ```
+
+1. Enable the Rate Limiting and Rate Limiting Advanced plugins for each tier:
+  ```yaml
+  plugins:
```

Comment:
Something isn't working in the formatting here when it's rendered.

Reply:
@cloudjumpercat I fixed the issues, and left an explanation in the commit message.
There's a caveat that I forgot to mention in the message though, and that is that if the content of a list item starts with a codeblock, there must be an empty line between the list item "title" and the code block. i.e.


```
1. Enable the Rate Limiting and Rate Limiting Advanced plugins for each tier:
    
   ```yaml # 3 spaces, aligned to Enable
          plugins:
   ```
```